mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSourceSchemaReader.java

```diff
@@ -71,12 +75,23 @@ public static boolean shouldConvertToDatetime(ResultSetMetaData metadata, int in
     // datetime and datetime2 will have type Types.TIMESTAMP
     // cannot decide based on sql type
     String columnTypeName = metadata.getColumnTypeName(index);
-    return shouldConvertToDatetime(columnTypeName);
```

Comment:
Remove the function shouldConvertToDatetime(String) and move the logic to the default of the switch block.

Reply:
removed